sdk/python/jobs/pipelines/1e_pipeline_with_registered_components/pipeline_with_registered_component_in_registry.ipynb

```diff
@@ -0,0 +1,248 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b02e37bd",
+   "metadata": {},
+   "source": [
+    "# 1. Connect to Azure Machine Learning Workspace"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ae915f2e",
+   "metadata": {},
+   "source": [
+    "## 1.1 Import the required libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6bebb0a7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.identity import DefaultAzureCredential, InteractiveBrowserCredential\n",
+    "\n",
+    "from azure.ai.ml import MLClient\n",
+    "from azure.ai.ml.dsl import pipeline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "54eaf508",
+   "metadata": {},
+   "source": [
+    "## 1.2 Configure credential"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "463f5692",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    credential = DefaultAzureCredential()\n",
+    "    # Check if given credential can get token successfully.\n",
+    "    credential.get_token(\"https://management.azure.com/.default\")\n",
+    "except Exception as ex:\n",
+    "    # Fall back to InteractiveBrowserCredential in case DefaultAzureCredential not work\n",
+    "    credential = InteractiveBrowserCredential()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fa8c5101",
+   "metadata": {},
+   "source": [
+    "## 1.3 Get a handle to the workspace and the registry"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3c246768",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get a handle to workspace\n",
+    "ml_client_ws = MLClient(\n",
```

Comment:
can also use mlclient from config, this will automagically work in integrated notebooks

Reply:
Yes, implemeneted like automl pipelines